staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/admission_test.go

```diff
@@ -423,6 +423,14 @@ func (c *celAdmissionController) getCurrentObject(obj runtime.Object) (runtime.O
 func waitForReconcile(ctx context.Context, controller *celAdmissionController, objects ...runtime.Object) error {
 	return wait.PollWithContext(ctx, 100*time.Millisecond, 1*time.Second, func(ctx context.Context) (done bool, err error) {
 		for _, obj := range objects {
+			defer func() {
```

Comment:
nit: does this deferred function need to be inside the loop, or do we want to run it once-per-call?

Reply:
thanks. definitely outside the loop